pnpm-lock.yaml

```diff
@@ -4517,7 +4517,7 @@ packages:
   /@automattic/viewport-react@1.0.0(react@17.0.2):
     resolution: {integrity: sha512-+6+l4jj14GXeoc5Jpic5E5eVvNL88Ezz8cMLmKAw0fpPDsz4gJv7o0hgShu0hjGjKTtBeUkBGfFWMCdRjZaVcA==}
     peerDependencies:
-      react: ^17.0.2
+      react: ^16.0.0
```

Comment:
I assume this was all calculated automatically, but some entries like this are a little surprising 🤔

Reply:
Good spot. Could it be a mistake/result of merging a branch where one branch was using an older version of React. 😬

Reply:
When I run `pnpm clean && pnpm install` on `trunk`, the lock file does not change. 

When I update the axios version, then run `pnpm install`, then change the axios version back and run `pnpm install` again, the changes to the lock file remain -- even though everything else is the same as on `trunk`. I'm not sure what's going on here.